js/views/composer.js

```diff
@@ -187,6 +208,22 @@ define(function(require) {
 				_this.saveDraft();
 			}, this.draftInterval);
 		},
+		toggleEditor: function()
```

Comment:
if 'text only' mode is selected, the other tools should be either greyed out or hidden